sigpack/demo_c++/sp_demo_101.cpp

```diff
@@ -1,3 +1,6 @@
+

```

Comment:
Rather than manually delete the changes and generate all these empy commits, you would be better off just checking out the folder from develop
```
git checkout develop sigpack/
git commit <message etc.>
```